docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+# Create Your First Tizen IoT .NET Application
+
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Tizen .NET application development process. With the instructions, you can create and run a basic .NET application, which displays some texts on the screen without any user interaction.
+
+1.  Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
+
+2. [Create a Project](#create-a-project) using Visual Studio.
+
+    This step shows how you can use a predesigned project template that creates all the basic files and folders required for your project.
+
+3. [Build Your Application](#build-your-application).
+
+    After you have implemented code for the features you want, this step shows how you can build the application to validate and compile
+    the code.
+
+4. [Deploy and Run Your Application](#deploy-and-run-your-application).
+
+    This step shows how you can deploy and run the application on the reference board.
+
+5. [Understand Your Application](#understand-your-application).
+
+    This step shows how you can enhance your application by creating a UI and making small alterations to improve the usability of the application.
```

Comment:
> This step shows how you can _enhance your application_ by _creating a UI_ and _making small alterations_ to improve the usability of the application.

I wish "Understand Your Application" section would have had more description about "creating a UI" or "making small alteration to improve the usability of the application".

For now the "Understand Your Application" seems to focus on describing the default/existing notion more, rather than "enhancing your application".

So, if you are not planning to enhance the "Understand Your Application" section (I think the section name is also changed from "enhance" to "understand"),
how about changing the description sentence instead, for this case?

For example, "This step shows the default structure of the basic files" or in this way.

Would you please share your suggestion?

Reply:
I update it based on your example.